### PR TITLE
QM9q: implement the reference energy computation

### PR DESCRIPTION
**Pull request by raimis**
- [x] Atomic energies
- [x] Reference energy computation for charged molecules

### CODE REVIEW

File: torchmdnet/datasets/qm9q.py [hunk 1]
```diff
@@ -11,6 +11,26 @@ class QM9q(Dataset):
...
+             1: -53.9895574739},
+        8: {-1: -74.9699154500,
+             0: -74.9812632126,
+             1: -74.4776884006},
+        9: {-1: -99.6695561536,
+             0: -99.6185158728},
+    }
+
+    INITIAL_CHARGES = {element: sorted([(energy, charge) for charge, energy in charges.items()])[0][1]
+                       for element, charges in ATOM_ENERGIES.items()}
```
Comment (stefdoerr) [inline torchmdnet/datasets/qm9q.py:32]: Avoid double nested list comprehensions please. They are quite impossible to read

Comment (raimis) [reply torchmdnet/datasets/qm9q.py]: Split the loops and added more docs

**Comment by raimis** [comment]
@stefdoerr can you finish reviewing?

**Review by stefdoerr — APPROVED** [review]
Looks good. I'm only curious if the charge atom energies are additive. Like does a +2 atom create equal energy increase as two +1 atoms?

**Pull request merged**